src/Microsoft.DotNet.Watcher.Tools/Program.cs

```diff
@@ -50,43 +69,31 @@ public static int Main(string[] args)
 
         private int MainInternal(string[] args, CancellationToken cancellationToken)
         {
-            var app = new CommandLineApplication();
-            app.Name = "dotnet-watch";
-            app.FullName = "Microsoft dotnet File Watcher";
-
-            app.HelpOption("-?|-h|--help");
+            var options = CommandLineOptions.Parse(args, _out);
+            if (options == null)
+            {
+                _error.WriteLine("Unexpected error parsing command line arguments");
```

Comment:
This is a "should never happen" condition...maybe I should use Debug.Assert instead...?